conda_forge_tick/migrators/arch.py

```diff
@@ -10,7 +10,7 @@
 from conda_forge_tick.utils import frozen_to_json_friendly, pluck, as_iterable
 from conda_forge_tick.xonsh_utils import indir
 from conda_forge_tick.make_graph import get_deps_from_outputs_lut
-
+from .migration_yaml import _all_noarch
```

Comment:
We may want to declare this a public API by removing the underscore.

Reply:
done